.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
     - env: DM=~5.0
       php: 5.6
     - env: DM=~4.2
-      php: 7
```

Comment:
Seems unlikely there will be a break that is on just 7.0. If you're worried about it, you can always add an extra build, but that makes the thing slower ofc.